lightweight-term-count-update.php

```diff
@@ -229,30 +229,33 @@ public function quick_update_terms_count( $object_id, $tt_ids, $taxonomy, $trans
 				// Ensure that these terms haven't already been counted.
 				$tt_ids = array_diff( $tt_ids, $this->counted_terms[ $object_id ][ $taxonomy ][ $transition_type ] );
 
-				if ( ! empty( $tt_ids ) ) {
```

Comment:
Why did `! empty( $tt_ids )` not work vs our change to `empty( $tt_ids )`

Reply:
Would moving `clean_term_cache( $tt_ids, $taxonomy, false );` inside the conditional fix the problem?

Reply:
Yes, it will solve the problem(that will be [here](https://github.com/Automattic/lightweight-term-count-update/blob/00a8ed8421b2c1435e3557719ea47ef6914ecab9/lightweight-term-count-update.php#L256)) but then we will need to add the same code inside [this](https://github.com/Automattic/lightweight-term-count-update/blob/00a8ed8421b2c1435e3557719ea47ef6914ecab9/lightweight-term-count-update.php#L222) if condition too. That will be like adding the same code to both `if` and `else` section.